canonical-web-and-design/ubuntu.com
# Update /advantage redirect

## Done

- Update /advantage redirect to /support from /pricing

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you end up on http://0.0.0.0:8001/support?utm_source=ua-client&utm_medium=cli


## Issue / Card

Fixes #5415
